apps/scan/frontend/src/screens/scan_error_screen.tsx

```diff
@@ -26,50 +27,117 @@ export function ScanErrorScreen({
   scannedBallotCount,
   restartRequired = false,
 }: Props): JSX.Element {
-  const errorMessage = (() => {
-    if (!error) return undefined;
-    assert(
-      error !== 'double_feed_calibration_timed_out' && // Only used in double feed calibration
-        error !== 'scanner_diagnostic_failed' && // Only used in ScannerDiagnosticScreen
-        error !== 'outfeed_blocked' // Only used in ScanJamScreen
-    );
+  assert(
+    error !== 'double_feed_calibration_timed_out' && // Only used in double feed calibration
+      error !== 'scanner_diagnostic_failed' && // Only used in ScannerDiagnosticScreen
+      error !== 'outfeed_blocked' // Only used in ScanJamScreen
+  );
+
+  const {
+    title,
+    errorMessage,
+    pollWorkerMessage,
+  }: {
+    title: JSX.Element;
+    errorMessage: JSX.Element;
+    pollWorkerMessage: 'needs-help' | 'might-need-help' | 'none';
+  } = (() => {
+    if (
+      restartRequired ||
+      !error ||
+      error === 'paper_status_timed_out' ||
+      error === 'scanning_timed_out' ||
+      error === 'unexpected_paper_status' ||
+      error === 'unexpected_event' ||
+      error === 'client_error'
+    ) {
+      return {
+        title: appStrings.titleScannerError(),
+        errorMessage: (
+          <Font weight="bold">
+            {appStrings.instructionsScannerAskForRestart()}
+          </Font>
+        ),
+        pollWorkerMessage: 'none',
```

Comment:
The restart screen is unique in that it doesn't have a poll worker message, which is instead baked into the error message. An alternative would be, instead of:

"The ballot was not counted. Ask a poll worker to restart the scanner."

It could be:

"The ballot was not counted. Scan it again after the scanner is restarted."
"Please ask a poll worker for help."

Do you think there is any reason for the restart screen to be special like it is?

Reply:
I think it's a little clearer the way it is now. The line about asking a poll worker for help is so small that I doubt anyone ever reads it.